### ISSUE TITLE
Dashboard?

### ISSUE THREAD

**Issue by KillaPL** [KillaPL/arch_app#4]
Logged in user might go to his dashboard, where he would see all recent actions he has not acknowledged, like new messages in available places.
We would, of course, need to keep track of these events.

**Comment by embargiel** [comment]
Good idea, but we should focus on main features firstly. Time tracking etc.

**Comment by KillaPL** [comment]
I consider dashboard with activity logging a main feature. It's unusable without it.
